### DISCUSSION TITLE
Is ent_entropy already normalized?

### DISCUSSION THREAD

**Discussion by thefacemyer** [QuSpin/QuSpin#660] [General]
I am calculating entanglement entropy for a subystem as a function of separation from the part of the system. I am getting a very peculiar result, and would like to know if there are any normalization factors built into the function that I'm not aware of? For example, is the von Neumann entropy (\alpha = 1) scaled by ln(2)?

**Comment by mgbukov** [comment]
The only normalization by definition is the density (i.e., the function
computes entanglement per spin). Check the default value of the
corresponding flag.
